aptos-move/block-executor/src/view.rs

```diff
@@ -1461,12 +1876,12 @@ impl<T: Transaction> ValueToIdentifierMapping for TemporaryExtractIdentifiersMap
 
 #[cfg(test)]
 mod test {
-    use super::LatestView;
+    use super::*;
```

Comment:
Let's add a big todo to refactor/revisit tests :)

Reply:
sure, we need to add resource group tests, etc.

but hopefully they are a bit better, cleaner and more comprehensive with this PR :)

Reply:
Done.